bin/parse_sdrf.py

```diff
@@ -9,18 +9,18 @@
 
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
+warnings = dict()
 
 @click.group(context_settings=CONTEXT_SETTINGS)
 def cli():
-  """This is the main tool that give access to all commands and options provided by the mslookup and dleamse algorithms"""
-
+  """Tool to convert sdrf files into OpenMS config files"""
 
 def openms_ify_mods(sdrf_mods):
   oms_mods = list()
 
   for m in sdrf_mods:
-    if "AC=UNIMOD" not in m:
-      raise Exception("only UNIMOD modifications supported.")
+    if "AC=UNIMOD" not in m and "AC=Unimod" not in m:
+      raise Exception("only UNIMOD modifications supported. " + m)
```

Comment:
https://www.python.org/dev/peps/pep-0498/

Can highly recommend (even more useful with several interpolations and more complex scenarios).

Reply:
True. Actually, this script comes from upstream. I was thinking about how to best track its changes from @timosachsenberg. But I think it should be in a completely finished state soon, such that we barely would need to update.

Reply:
https://github.com/bigbio/sdrf-openms/blob/master/parse_sdrf.py

Reply:
Well you can always just package it and put it on PyPi & Conda. Might be a little bit overkill for such a small script, but would help.

Reply:
Ok thanks for the Tipp! I will take a look at it.